dissect/cstruct/parser.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, cs: cstruct, compiled: bool = True, align: bool = False):
         self.compiled = compiled
         self.align = align
         self.TOK = self._tokencollection()
+        self.cstruct.includes = []
```

Comment:
I meant in the `__init__` of the `cstruct` class.

Reply:
[0a2a0f8](https://github.com/fox-it/dissect.cstruct/pull/129/commits/0a2a0f8aaf0bb5d0b4eb1b932955c45c14dbe71a)